cellprofiler/modules/measureimageareaoccupied.py

```diff
@@ -108,9 +108,9 @@ def __init__(self):
                 self.__operand_objects = cellprofiler.setting.ObjectNameSubscriber(
                     "Select objects to measure",
                     cellprofiler.setting.NONE,
-                    doc="""
-                    <i>(Used only if '{O_OBJECTS}' are to be measured)</i><br>
-                    Select the previously identified objects you would like to measure.
+                    doc="""\
+*(Used only if ‘{O_OBJECTS}’ are to be measured)*
```

Comment:
Add a newline after *(Used only if...)* text to insert a line break in the documentation rendered in the GUI.